lib/engagement.js

```diff
@@ -16,6 +16,19 @@ class Engagement {
     }, cb)
   }
 
+  associatedEngagements (objectType, objectId, options, cb) {
```

Comment:
Hi @jayprakash1, thanks for the contribution.

This doesn't quite match the naming convention for methods. I'm not familiar with this particular API call, but what about `getAssociated`?